lib/Model/Project.php

```diff
@@ -5,57 +5,66 @@
 namespace Doctrine\Website\Model;
 
 use Closure;
-use Doctrine\SkeletonMapper\Mapping\ClassMetadataInterface;
-use Doctrine\SkeletonMapper\Mapping\LoadMetadataInterface;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\Website\Repositories\ProjectRepository;
 use InvalidArgumentException;
 
 use function array_filter;
 use function array_values;
 use function sprintf;
 
-class Project implements LoadMetadataInterface
+#[ORM\Entity(repositoryClass: ProjectRepository::class)]
+class Project
 {
-    private ProjectIntegrationType|null $projectIntegrationType = null;
-
-    private ProjectStats $projectStats;
-
-    private bool $active;
-
-    private bool $archived;
-
-    private string $name;
-
-    private string $shortName;
-
-    private string $slug;
-
-    private string $docsSlug;
-
-    private string $composerPackageName;
-
-    private string $repositoryName;
-
-    private bool $isIntegration = false;
-
-    private string $integrationFor;
-
-    private string $docsRepositoryName;
-
-    private string $docsPath;
-
-    private string $codePath;
-
-    private string $description;
-
-    /** @var string[] */
-    private array $keywords = [];
-
-    /** @var ProjectVersion[] */
-    private array $versions = [];
-
-    public static function loadMetadata(ClassMetadataInterface $metadata): void
-    {
-        $metadata->setIdentifier(['slug']);
+    /**
+     * @param Collection<int, ProjectVersion> $versions
+     * @param string[]                        $keywords
+     */
+    public function __construct(
+        #[ORM\OneToOne(targetEntity: ProjectStats::class, fetch: 'EAGER')]
```

Comment:
I chose `EAGER` because every piece of data is used in the website build and I didn't experienced a decrease in speed. Maybe the build time can be compared with the ones from `develop` from the daily live deployments.